### PR TITLE
fix: resolve user implemented mappings correctly in queryable projections

### PR DESCRIPTION
**Pull request by latonz**

### CODE REVIEW

File: src/Riok.Mapperly/Descriptors/InlineExpressionMappingBuilderContext.cs [hunk 1]
```diff
@@ -67,8 +67,8 @@ public override bool IsConversionEnabled(MappingConversionType conversionType)
         if (_inlineExpressionMappings.Find(sourceType, targetType) is { } mapping)
             return mapping;
 
-        // user implemented mappings are also taken into account
-        // this works as long as the user implemented methods
+        // user implemented mappings are also taken into account.
```
Comment (ni507) [inline src/Riok.Mapperly/Descriptors/InlineExpressionMappingBuilderContext.cs:70]: `User` should be uppercase.

File: test/Riok.Mapperly.IntegrationTests/Models/TestObjectProjection.cs [hunk 2]
```diff
@@ -5,9 +5,11 @@ namespace Riok.Mapperly.IntegrationTests.Models
 {
     public class TestObjectProjection
     {
+        public int Id { get; set; }
+
         public int CtorValue { get; set; }
 
-        public int IntValue { get; set; }
+        public int Intvalue { get; set; }
```
Comment (ni507) [inline test/Riok.Mapperly.IntegrationTests/Models/TestObjectProjection.cs:12]: This can be removed?

**Review by ni507 — CHANGES_REQUESTED** [review]

File: test/Riok.Mapperly.IntegrationTests/Models/TestObjectProjection.cs (hunk 2 above)
Comment (latonz) [reply test/Riok.Mapperly.IntegrationTests/Models/TestObjectProjection.cs:12]: Fixed the typo but keep it to ensure it is correctly mapped.

File: src/Riok.Mapperly/Descriptors/InlineExpressionMappingBuilderContext.cs (hunk 1 above)
Comment (latonz) [reply src/Riok.Mapperly/Descriptors/InlineExpressionMappingBuilderContext.cs]: Fixed.

**Review by ni507 — APPROVED** [review]

**Pull request merged**